.github/ami-cleaner.sh

```diff
@@ -0,0 +1,57 @@
+DO_CLEANUP=${DO_CLEANUP:-false}
+AMI_OWNER=${AMI_OWNER:-"053594193760"}
+MAX_AMI_NUMBER=${MAX_AMI_NUMBER:-20}
+
+
+regions=( $( aws ec2 describe-regions | jq '.Regions[].RegionName' -r ) )
+
+
+
+echo "------------------------------------------------------------------"
+if [[ "${DO_CLEANUP}"  == "true" ]]; then
+  printf "\xF0\x9F\x92\x80 Cleaning mode activated\n"
+else
+  printf "\xF0\x9F\x93\x84 Reporting mode activated\n"
+fi
+echo "Maximun number of AMIs allowed: $MAX_AMI_NUMBER"
+echo "Owner of the AMIs: $AMI_OWNER"
+echo "Regions: ${regions[@]}"
+echo "------------------------------------------------------------------"
+
+touch removed.txt
+
+
+for region in "${regions[@]}"; do
+  printf "\xE2\x86\x92 Checking AMIS on region %s for owner %s\n" "${region}" "${AMI_OWNER}"
+  export AWS_DEFAULT_REGION=${region}
+  ami_number=$( aws ec2 describe-images --owners $AMI_OWNER | jq '.Images| length' )
+  snapshots_number=$( aws ec2 describe-snapshots --owner-id $AMI_OWNER | jq '.Snapshots| length' )
+  printf "\xE2\x86\x92 Found %s AMIs\n" "$ami_number"
+  printf "\xE2\x86\x92 Found %s Snapshots\n" "$snapshots_number"
+  if [[ $ami_number > $MAX_AMI_NUMBER ]]; then
+    printf "\xE2\x86\x92 AMI number is bigger than max number allowed, engaging cleanup.\n"
+    to_clean=$(expr $ami_number - $MAX_AMI_NUMBER)
+    printf "\xE2\x86\x92 Cleaning %s AMIs+Snapshots\n" "$to_clean"
+    to_clean_amis=$(aws ec2 describe-images --owners 053594193760 |jq -r ".Images | sort_by(.CreationDate) | .[:${to_clean}] | .[]| {ImageId: .ImageId, CreationDate: .CreationDate, SnapshotId: .BlockDeviceMappings[].Ebs.SnapshotId}")
```

Comment:
Looks like this should consume the `AMI_OWNER` variable instead of the literal

Reply:
Wooops, good catch